solara/website/components/sidebar.py

```diff
@@ -34,58 +37,61 @@ def Sidebar():
                                             else "/apps/" + subitem.path
                                         )
                                         with solara.Link(
                                             path,
                                         ):
-                                            with solara.v.ListItem(dense=True, style_="padding: 0 20px;", value=path):
+                                            with solara.v.ListItem(dense=True, style_="padding-left: 60px; padding-right: 20px", value=path):
                                                 solara.v.ListItemContent(
                                                     children=[subitem.label],
                                                 )
                             else:
-                                with solara.v.ListItemGroup(value="/documentation/" + route.path + "/" + item.path):
-                                    with solara.Link(
-                                        "/documentation/" + route.path + "/" + item.path,
-                                    ):
-                                        with solara.v.ListItem(dense=True, style_="padding: 0 20px;"):
-                                            solara.v.ListItemContent(
-                                                children=[item.label],
-                                            )
-        with solara.v.ListItemGroup():
-            with solara.Link("/contact"):
-                with solara.v.ListItem():
-                    solara.v.ListItemIcon(children=[solara.v.Icon(children=["mdi-email"])])
-                    solara.v.ListItemTitle(style_="padding: 0 20px;", children=["Contact"])
-            with solara.Link("/changelog"):
-                with solara.v.ListItem():
-                    solara.v.ListItemIcon(children=[solara.v.Icon(children=["mdi-history"])])
-                    solara.v.ListItemTitle(style_="padding: 0 20px;", children=["Changelog"])
+                                path = "/documentation/" + route.path + ("/" + item.path if item.path != "/" else "")
+                                with solara.Link(path):
+                                    with solara.v.ListItem(dense=True, style_="padding: 0 20px;", value=path):
+                                        solara.v.ListItemContent(
+                                            children=[label],
+                                        )
+        with solara.Link("/contact"):
+            with solara.v.ListItem():
+                solara.v.ListItemIcon(children=[solara.v.Icon(children=["mdi-email"])])
+                solara.v.ListItemTitle(style_="padding: 0 20px;", children=["Contact"])
+        with solara.Link("/changelog"):
+            with solara.v.ListItem():
+                solara.v.ListItemIcon(children=[solara.v.Icon(children=["mdi-history"])])
+                solara.v.ListItemTitle(style_="padding: 0 20px;", children=["Changelog"])
```

Comment:
```suggestion
            with solara.Link("/contact"):
                with solara.v.ListItem(value="/contact"):
                    solara.v.ListItemIcon(children=[solara.v.Icon(children=["mdi-email"])])
                    solara.v.ListItemTitle(style_="padding: 0 20px;", children=["Contact"])
            with solara.Link("/changelog"):
                with solara.v.ListItem(value="/changelog"):
                    solara.v.ListItemIcon(children=[solara.v.Icon(children=["mdi-history"])])
                    solara.v.ListItemTitle(style_="padding: 0 20px;", children=["Changelog"])
```

Because these items are now no longer in the `ListItemGroup`, they are not highlighted when you're viewing them. You also don't get the background colour change when hovering over them.